packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -480,6 +481,69 @@ const Conversation = WebexPlugin.extend({
       });
   },
 
+  /**
+   * Paginates through a set of conversations. By default does not fetch activities or
+   * participants
+   * @param {Object} options
```

Comment:
Could you add what the `options.page` object is supposed to be?

Reply:
Added